GetJobber/atlantis
# Job 61137 fix(components) Emphasis Svg not included in build

## Motivations



## Changes



### Added

- 

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- Inline the Emphasis SVG so it is included in components release builds

### Security

- 

## Testing

- Install new dependencies.
- Run a npm build.
- Navigate to `dist/typography-<numbers>.js
- Observe there is an inline SVG background image for the emphasis class

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

![Random photo of Atlantis](https://loremflickr.com/672/400/atlantis)
